### DISCUSSION TITLE
it/trentino-trasporti-digitale-2/

### DISCUSSION THREAD

**Discussion by giscus** [matteocontrini/blog#13] [Comments]
# it/trentino-trasporti-digitale-2/

Venti mesi dopo, cosa è cambiato? Con qualche riflessione sull’uso del digitale per semplificare l’uso del trasporto pubblico locale.

https://matteosonoio.it/it/trentino-trasporti-digitale-2/

**Comment by fcucino** [comment]
Inoltre gli screenshot del sito di TT immortalano l'icona della linea `5/` rotta perché contiene proprio lo `/` nell'url

**Comment by gwabbey** [comment]
wow, ho sempre pensato i QR fossero riferimenti interni, e invece... comunque è assurdo come non riescano a migliorare i loro servizi digitali, soprattutto l'app
